src/backend/api/meals.js

```diff
@@ -4,9 +4,132 @@ const knex = require("../database");
 
 router.get("/", async (request, response) => {
   try {
-    // knex syntax for selecting things. Look up the documentation for knex for further info
-    const titles = await knex("meals").select("title");
-    response.json(titles);
+    //to get meals that has a price smaller than maxPrice
+    if ('maxPrice' in request.query) {
+      const maxPrice = parseInt(request.query.maxPrice);
+      if (isNaN(maxPrice)) {
+        response.status(400).send({ error: "maxPrice should be an integer" });
+        return;
+      }
+      const mealsLessThanMaxPrice = await knex("meal").where('price', '<', maxPrice);
```

Comment:
```suggestion
      const mealsLessThanMaxPrice = await knex("meals").where('price', '<', maxPrice);
```
i think the name of the database is `meals` and not `meal`

Reply:
hi florin, the table name is meal